src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1594,6 +1594,20 @@ else if (ModElementType.FEATURE.equals(modElement.getType()) && uiTest) {
 			feature.featurexml = Feature.XML_BASE;
 			feature.skipPlacement = !_true;
 			return feature;
+		} else if (ModElementType.ATTRIBUTE.equals(modElement.getType())) {
+			Attribute attribute = new Attribute(modElement);
+			attribute.name = modElement.getName();
+			attribute.defaultValue = 127.53;
+			attribute.minValue = 24.42;
+			attribute.maxValue = 200d;
+			attribute.entities = new ArrayList<>();
+			if (!emptyLists) {
+				attribute.entities.addAll(ElementUtil.loadAllEntities(modElement.getWorkspace()).stream().map(e -> new net.mcreator.element.parts.EntityEntry(modElement.getWorkspace(), e.getName()))

```

Comment:
I would assume you would want loadAllSpawnableEntities here since UI also only supports that now